docs/source/installation_guide.md

```diff
@@ -82,6 +82,24 @@ pip install neural-compressor-tf
   [optional] pip install -r requirements_tf.txt # for TensorFlow framework extension API
   ```
 
+- [PyTorch only]
+    ```Shell
+    git clone https://github.com/intel/neural-compressor.git
+    cd neural-compressor
+    pip install -r requirements_pt.txt # for PyTorch framework extension API
+    export INC_PT_ONLY=1
+    pip install -e .
```

Comment:
should be `pip install .` ?

Reply:
`-e` is also popular used in many repos, such as [torchao](https://github.com/pytorch/ao?tab=readme-ov-file#installation). Since it's not a blocking issue, let's keep it same as the internal repo.